src/components/AddressSearch.js

```diff
@@ -23,134 +23,127 @@ const propTypes = {
-            onPress={(data, details) => {
-                saveLocationDetails(details);
-
-                // After we select an option, we set displayListViewBorder to false to prevent UI flickering
-                setDisplayListViewBorder(false);
-            }}
-            query={{
-                key: 'AIzaSyC4axhhXtpiS-WozJEsmlL3Kg3kXucbZus',
-                language: props.preferredLocale,
-                types: 'address',
-                components: 'country:us',
-            }}
-            requestUrl={{
-                useOnPlatform: 'web',
-                url: `${CONFIG.EXPENSIFY.URL_EXPENSIFY_COM}api?command=Proxy_GooglePlaces&proxyUrl=`,
-            }}
-            textInputProps={{
-                InputComp: ExpensiTextInput,
-                label: props.label,
-                containerStyles: props.containerStyles,
-                errorText: props.errorText,
-                onChangeText: (text) => {
-                    const isTextValid = !_.isEmpty(text) && _.isEqual(text, props.value);
-
-                    // Ensure whether an address is selected already or has address value initialized.
-                    if (!_.isEmpty(googlePlacesRef.current.getAddressText()) && !isTextValid) {
-                        saveLocationDetails({});
-                    }
-
-                    // If the text is empty, we set displayListViewBorder to false to prevent UI flickering
-                    if (_.isEmpty(text)) {
-                        setDisplayListViewBorder(false);
-                    }
-                },
-            }}
-            styles={{
-                textInputContainer: [styles.flexColumn],
-                listView: [
-                    !displayListViewBorder && styles.googleListView,
-                    displayListViewBorder && styles.borderTopRounded,
-                    displayListViewBorder && styles.borderBottomRounded,
-                    displayListViewBorder && styles.mt1,
-                    styles.overflowAuto,
-                    styles.borderLeft,
-                    styles.borderRight,
-                ],
-                row: [
-                    styles.pv4,
-                    styles.ph3,
-                    styles.overflowAuto,
-                ],
-                description: [styles.googleSearchText],
-                separator: [styles.googleSearchSeparator],
-            }}
-            onLayout={(event) => {
-                // We use the height of the element to determine if we should hide the border of the listView dropdown
-                // to prevent a lingering border when there are no address suggestions.
-                // The height of the empty element is 2px (1px height for each top and bottom borders)
-                setDisplayListViewBorder(event.nativeEvent.layout.height > 2);
-            }}
-        />
-    );
-};
+    }
+
+    render() {
+        return (
+            <GooglePlacesAutocomplete
+                fetchDetails
```

Comment:
nab: my IDE is complaining about no `placeholder` required attribute set

Reply:
You are right, there is a mandatory prop "placeholder" in `GooglePlacesAutocomplete` ([see here](https://github.com/FaridSafi/react-native-google-places-autocomplete/blob/47d7223dd48f85da97e80a0729a985bbbcee353f/GooglePlacesAutocomplete.d.ts#L409)). We haven't been using it.

If I add `placeholder="PLACEHOLDER"`, it looks like this:

https://user-images.githubusercontent.com/87341702/142874673-d288ae2e-8473-4768-a20b-a16ab0d5775c.mov

We could get rid of the error by just doing `placeholder=""` if we don't want to have a placeholder text.

Reply:
Was going to suggest just doing `placeholder=""`, I think that'd be best